.github/workflows/build_and_publish.yml

```diff
@@ -49,4 +49,24 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: built-grain-wheels-${{ matrix.os }}-${{ matrix.python-version }}
-          path: /tmp/grain/all_dist/**/*.whl
\ No newline at end of file
+          path: /tmp/grain/all_dist/*.whl
+
+  publish-wheel:
+    runs-on: ubuntu-22.04
+    needs: build-and-test
+    permissions:
+      id-token: write
+    environment:
+      name: pypi
+      url: https://pypi.org/project/grain
+    steps:
+      - name: Download Grain artifacts
+        uses: actions/download-artifact@v4
+        with:
+          pattern: built-grain-wheels-*
```

Comment:
I think for linux the artifacts include audited wheels (manylinux) and the non-audited ones (jist linux suffix). Could you pls change this to only copy the audited ones:
https://github.com/google/grain/blob/56bc5793de84282b3955dd4ba8d78184de7af1f7/grain/oss/build_whl.sh#L85

Reply:
I added another step, "Remove unaudited wheels", which removes that one on linux (It isn't used anywhere after all).

To test this PR we could have 0.2.10b1 for example. For me to test it I would need access to PyPI account to configure [trusted publishing](https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/#configuring-trusted-publishing) and allow `build_and_publish.yml` to push wheels to PyPI.